blockchain/blockchain-wallet-v4-frontend
# Fix/testnet support

## TODO
- [x] Fix kvstore entries
- [x] Fix send
- [x] Fix add wallet account
- [ ] Disable other currencies?

## Description
Pull in network from wallet-options

## Change Type
- Feature
Added testnet support

- Bug Fix
~~1. The var `network` is assigned with `settings.NETWORK_BITCOIN` but we are using `NETWORK` as the name in `config/index.js` so its value is always `undefined`. Renamed `NETWORK` as `NETWORK_BITCOIN`
2. Passing `network` as a param in `actions.auth.register()` function~~

## Testing Steps
1. Change wallet-options-v4 in config/ to use testnet instead of bitcoin network.
2. Run `yarn start:testnet`
3. Should successfully create a wallet

## Code Checklist
